newIDE/app/src/SceneEditor/index.js

```diff
@@ -514,7 +517,14 @@ export default class SceneEditor extends React.Component<Props, State> {
   };
 
   _onInstancesAdded = (instances: Array<gdInitialInstance>) => {
+    const invisibleLayersOnWhichInstancesHaveJustBeenAdded = [];
     instances.forEach(instance => {
+      const layer = this.props.layout.getLayer(instance.getLayer());
+      if (!layer.getVisibility()) {
+        invisibleLayersOnWhichInstancesHaveJustBeenAdded.push(
```

Comment:
is there a risk you add the same layer multiple times in this array?

Reply:
Good question!